pkg/storage/unified/sql/db/migrations/resource_mig.go

```diff
@@ -49,6 +49,7 @@ func initResourceTables(mg *migrator.Migrator) string {
 			{Name: "name", Type: migrator.DB_NVarchar, Length: 190, Nullable: false},
 			{Name: "value", Type: migrator.DB_LongText, Nullable: true},
 			{Name: "action", Type: migrator.DB_Int, Nullable: false}, // 1: create, 2: update, 3: delete
+			{Name: "previous_resource_version", Type: migrator.DB_BigInt, Nullable: true},
```

Comment:
Q: Haven't done a DB migration yet. Is this how It's supposed to happen ?

Reply:
Never added a migration either, but by looking at the [Dashboards migrations](https://github.com/grafana/grafana/blob/main/pkg/services/sqlstore/migrations/dashboard_mig.go#L74-L88) seems like we need to make a new call to `AddMigration` with the new desired structure.